src/components/Dao/DaoLandingPage.tsx

```diff
@@ -42,7 +42,7 @@ export default class DaoLandingPage extends React.Component<IProps, null> {
 
   public render() {
     const daoState = this.props.daoState;
-    const customData = customDaoInfo[process.env.NETWORK][daoState.id.toLowerCase()];
+    const customData = customDaoInfo[process.env.NETWORK]?.[daoState.id.toLowerCase()];
```

Comment:
how and why this is manifested only in ganache ?

Reply:
happens when "ganache" is called "private".  Your question made me realize that we should be using `targetedNetwork` instead of `process.env.NETWORK`.  So I changed that in this PR, and will also change it in the 1=>2 merge PR.

Reply:
so this is manifested on all other networks?

Reply:
please do not add that to the merge to 2.0 pr..
you can pr that separately if you wish .